### ISSUE TITLE
nrk-switch

### ISSUE THREAD

**Issue by eirikbacker** [nrkno/core-css#65]
Origo-CSS has a nice switch-design, would be nice to expose this in core-css; https://static.nrk.no/origo-css/latest/index.html?forms.md#switches

**Comment by htor** [comment]
what is the use case for switches vs. checboxes? should the switch be a styled checkbox, or a button? should it auto-submit?

**Comment by eirikbacker** [comment]
```
.org-label input[type="checkbox"] {
  cursor: pointer;
  width: 5em;
  height: 2.5em;
  border-radius: 999px;
  box-shadow: inset -2.5em 0 0 2px, inset 0 0 0 2px;
  transition: .2s;
  opacity: .5;
}

.org-label input[type="checkbox"]:hover { opacity: .7 }
.org-label input[type="checkbox"]:active:hover { box-shadow: inset -1.5em 0 0 2px, inset 0 0 0 2px }
.org-label input[type="checkbox"].org-switch--s { font-size: var(--org-text-s) }
.org-label input[type="checkbox"].focus-visible { opacity: 1; border: 2px solid var(--org-color-focus) }
.org-label input[type="checkbox"]:disabled { opacity: .2 }
.org-label input[type="checkbox"]:checked:active:hover { box-shadow: inset 1.5em 0 0 2px, inset 0 0 0 2px }
.org-label input[type="checkbox"]:checked:checked {
  opacity: 1;
  color: var(--org-color-primary);
  box-shadow: inset 2.5em 0 0 2px, inset 0 0 0 2px;
}
```

**Comment by eirikbacker** [comment]
closed by #66
